templates/admin.php

```diff
@@ -0,0 +1,23 @@
+<?php
+//	script('facerecognition', 'admin');
+?>
+
+<form id="facerecognition">
+	<div class="section">
+		<h2>
+			<?php p($l->t('Face recognition'));?>
+			<a target="_blank" rel="noreferrer noopener" class="icon-info" title="<?php p($l->t('Open Documentation'));?>" href="https://github.com/matiasdelellis/facerecognition/wiki"></a>
+		</h2>
+		<h3>Configuration information</h3>
+		<p><strong>Requirements: </strong><?php p($_['msg']); ?><span class="status success<?php if(!$_['requirements']):?> error<?php endif;?>"></span></p>
+		<h3>Models</h3>
+		<p><strong>Recognition Model: </strong><?php p($_['recognition-model']); ?></p>
```

Comment:
Consider (not required, just think about it):
Maybe we should not give user full path to models and what each model does. Maybe it is enough to just tell them something like "Using model: "1 (default)" and whether model files are present or not (as requirements). This is because it is unnecesarry information for user (why they need to know location of model files?) and maybe model "2" will not be having same set of files, maybe model 2 have "face_recognition", "baby_recognition", "nose_detector"... Again, whatever is easier/makes sense to you.

Reply:
You are right..
Considering that we now distribute the models in the application, it does not make sense to say what these models are.